scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -35,6 +35,7 @@ workflow case_gatk_acnv_workflow {
     String is_disable_reference_validation
     String jni_lib    
     String plots_dir
+    Boolean disable_gc
```

Comment:
This would be better to phrase in terms of the opposite `Boolean`, `enable_gc_correction`.